config/default.yml

```diff
@@ -489,6 +489,9 @@ Layout/SpaceBeforeBlockBraces:
   SupportedStyles:
     - space
     - no_space
+  SupportedStylesForEmptyBraces:
```

Comment:
It looks like you are missing setting a default value for `EnforcedStyleForEmptyBraces`. Based on the other usages, it looks like it should be set to `no_space`.